### PR TITLE
Add Bobcat Miner 300 to Hotspots App

### PR DESCRIPTION
**Pull request by JackeyTY**
fix merge conflict

### CODE REVIEW

File: src/features/hotspots/setup/HotspotSelectionScreen.tsx [hunk 1]
```diff
@@ -28,7 +28,8 @@ const HotspotSetupSelectionScreen = () => {
           <HotspotSelectionCard hotspotType="NEBRAIN" />
           <HotspotSelectionCard hotspotType="NEBRAOUT" />
         </Box>
-        <Box flexDirection="row" width="50%" height={191}>
+        <Box flexDirection="row" height={191}>
+          <HotspotSelectionCard hotspotType="Bobcat" />
```
Comment (tyler-whitman) [inline src/features/hotspots/setup/HotspotSelectionScreen.tsx:32]: Can you move this to after `SYNCROBIT`. They should remain in the order they were added.

Comment (JackeyTY) [reply src/features/hotspots/setup/HotspotSelectionScreen.tsx:32]: Got it.

Comment (JackeyTY) [reply src/features/hotspots/setup/HotspotSelectionScreen.tsx]: Fixed

Comment (JackeyTY) [reply src/features/hotspots/setup/HotspotSelectionScreen.tsx]: I am not sure about the svg file, can you see it display properly on the selection screen?

Comment (tyler-whitman) [reply src/features/hotspots/setup/HotspotSelectionScreen.tsx]: Thanks, yes the SVG is working as expected

**Review by tyler-whitman — APPROVED** [review]
Looks good and working as expected. I will wait for one other to review and we can merge this in.

**Review by allenan — APPROVED** [review]

**Comment by JackeyTY** [comment]
Cool! Thank you team!

**Pull request merged**